editor/src/components/canvas/controls/property-target-selector.tsx

```diff
@@ -26,17 +27,17 @@ interface PropertyTargetSelectorProps {
 
 function labelForOption(option: LayoutTargetableProp): string {
```

Comment:
Looks like this function is now redundant

Reply:
Ha, so it is.

Reply:
Fixed in 7585ad850669034f876221620ecc02d4879ec51c.